docs/proposals/persistent-storage-provisioning.md

```diff
@@ -0,0 +1,58 @@
+# Persistent Storage Provisioning
+
+This document proposes a design taking advantage of cloud provider APIs to automate provisioning of persistent storage.
+
+
+## Background
+
+### Kubernetes Infrastructure (current and under development).
+
+Kubernetes currently has a mechanism for users to
+[request storage](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/persistent-storage.md#request-storage)
+for their pods, and a mechanism for admins to
+[describe the storage](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/persistent-storage.md#describe-available-storage)
+they have created and made available for use by pods. The various
+mechanisms for mounting the provided storage are provided by plugins
+such as the
+[GCE PD](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubelet/volume/gce_pd/gce_pd.go),
+[NFS plugin](https://github.com/GoogleCloudPlatform/kubernetes/pull/4601),
+the [iSCSI plugin](https://github.com/GoogleCloudPlatform/kubernetes/pull/4612)
+
+For more detail please see the [Persistent Storage design document](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/persistent-storage.md)
+
+### Cloud Provider APIs
+
+Cloud providers usually provide an API for requesting storage dynamically.
+
+For example:
+
+GCE: https://cloud.google.com/compute/docs/reference/latest/disks/insert
+
+AWS: http://docs.aws.amazon.com/AmazonS3/latest/API/SOAPCreateBucket.html
```

Comment:
The AWS service for block storage is EBS: http://aws.amazon.com/ebs/

(S3 backed storage would be interesting, but I don't think this is what you mean)